### PR TITLE
Applied fixes from StyleCI

### PR DESCRIPTION
**Pull request by lsmith77**
This pull request applies code style fixes from an analysis carried out by [StyleCI](https://styleci.io).

---

For more information, click [here](https://styleci.io/analyses/Xpgy28).

### CODE REVIEW

File: Controller/Annotations/View.php [hunk 1]
```diff
@@ -93,7 +95,7 @@ public function getStatusCode()
     }
 
     /**
-     * @var array $serializerGroups
+     * @var array
```
Comment (lsmith77) [inline Controller/Annotations/View.php]: I guess this was the wrong annotation to begin with

File: DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -42,7 +42,12 @@ public function getConfigTreeBuilder()
                 ->arrayNode('access_denied_listener')
                     ->canBeEnabled()
                     ->beforeNormalization()
-                        ->ifArray()->then(function($v) { if (!empty($v) && empty($v['formats'])) { unset($v['enabled']); $v = array('enabled' => true, 'formats' => $v); } return $v; })
+                        ->ifArray()->then(function ($v) { if (!empty($v) && empty($v['formats'])) {
+     unset($v['enabled']);
```
Comment (lsmith77) [inline DependencyInjection/Configuration.php]: hmm not sure that I like this without intention

File: Tests/Fixtures/Controller/AnnotatedConditionalUsersController.php [hunk 3]
```diff
@@ -11,137 +11,185 @@
...
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
 class AnnotatedConditionalUsersController extends Controller
 {
     public function optionsUsersAction()
-    {} // [OPTIONS]     /users
+    {
+    }
+
+ // [OPTIONS]     /users
```
Comment (lsmith77) [inline Tests/Fixtures/Controller/AnnotatedConditionalUsersController.php]: comments in this file need to be adjusted

File: Tests/Fixtures/Controller/AnnotatedNonPluralizedArticleController.php [hunk 4]
```diff
@@ -20,14 +20,26 @@
 class AnnotatedNonPluralizedArticleController extends FosRestController
 {
     public function cgetAction()
-    {} // [GET] /article
+    {
+    }
+
+ // [GET] /article
```
Comment (lsmith77) [inline Tests/Fixtures/Controller/AnnotatedNonPluralizedArticleController.php]: comments in this file need to be adjusted

File: Tests/Fixtures/Controller/AnnotatedPrefixedController.php [hunk 5]
```diff
@@ -22,5 +21,8 @@
 class AnnotatedPrefixedController extends Controller
 {
     public function getSomethingAction()
-    {} // [GET]     /aprefix/something.{_format}
+    {
+    }
+
+ // [GET]     /aprefix/something.{_format}
```
Comment (lsmith77) [inline Tests/Fixtures/Controller/AnnotatedPrefixedController.php]: comments in this file need to be adjusted

File: Tests/Fixtures/Controller/ArticleController.php [hunk 6]
```diff
@@ -11,88 +11,163 @@
...
 use Symfony\Component\HttpFoundation\Request;
 
 class ArticleController extends FosRestController implements ClassResourceInterface
 {
     public function optionsAction()
-    {} // [OPTIONS] /articles
+    {
+    }
+
+ // [OPTIONS] /articles
```
Comment (lsmith77) [inline Tests/Fixtures/Controller/ArticleController.php]: comments in this file need to be adjusted

File: Tests/Fixtures/Controller/AnnotatedUsersController.php [hunk 7]
```diff
@@ -11,146 +11,195 @@
...
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
 class AnnotatedUsersController extends Controller
 {
     public function optionsUsersAction()
-    {} // [OPTIONS]     /users
+    {
+    }
+
+ // [OPTIONS]     /users
```
Comment (lsmith77) [inline Tests/Fixtures/Controller/AnnotatedUsersController.php]: comments in this file need to be adjusted

File: Tests/Fixtures/Controller/InformationController.php [hunk 8]
```diff
@@ -17,5 +17,8 @@
 class InformationController extends FOSRestController implements ClassResourceInterface
 {
     public function cgetAction()
-    {} // [GET] /information
+    {
+    }
+
+ // [GET] /information
```
Comment (lsmith77) [inline Tests/Fixtures/Controller/InformationController.php]: comments in this file need to be adjusted

File: Tests/Fixtures/Controller/MediaController.php [hunk 9]
```diff
@@ -17,8 +17,14 @@
 class MediaController extends FOSRestController implements ClassResourceInterface
 {
     public function cgetAction()
-    {} // [GET] /media
+    {
+    }
+
+ // [GET] /media
```
Comment (lsmith77) [inline Tests/Fixtures/Controller/MediaController.php]: comments in this file need to be adjusted

File: Tests/Fixtures/Controller/OrdersController.php [hunk 10]
```diff
@@ -18,19 +18,34 @@ class OrdersController extends Controller
     // conventional HATEOAS action after REST action
 
     public function getFoosAction()
-    {} // [GET] /foos
+    {
+    }
+
+ // [GET] /foos
```
Comment (lsmith77) [inline Tests/Fixtures/Controller/OrdersController.php]: comments in this file need to be adjusted

File: Tests/Fixtures/Controller/UserTopicCommentsController.php [hunk 11]
```diff
@@ -16,23 +16,40 @@
 class UserTopicCommentsController extends Controller
 {
     public function getCommentsAction($slug, $title)
-    {} // [GET] /users/{slug}/topics/{title}/comments
+    {
+    }
+
+ // [GET] /users/{slug}/topics/{title}/comments
```
Comment (lsmith77) [inline Tests/Fixtures/Controller/UserTopicCommentsController.php]: comments in this file need to be adjusted

File: Tests/Fixtures/Controller/UserTopicsController.php [hunk 12]
```diff
@@ -16,25 +16,46 @@
 class UserTopicsController extends Controller
 {
     public function getTopicsAction($slug)
-    {} // [GET] /users/{slug}/topics
+    {
+    }
+
+ // [GET] /users/{slug}/topics
```
Comment (lsmith77) [inline Tests/Fixtures/Controller/UserTopicsController.php]: comments in this file need to be adjusted

File: Tests/Fixtures/Controller/UsersController.php [hunk 13]
```diff
@@ -17,87 +17,168 @@
 class UsersController extends Controller
 {
     public function optionsUsersAction()
-    {} // [OPTION] /users
+    {
+    }
+
+ // [OPTION] /users
```
Comment (lsmith77) [inline Tests/Fixtures/Controller/UsersController.php]: comments in this file need to be adjusted

File: DependencyInjection/Configuration.php (hunk 2 above)
Comment (stof) [inline DependencyInjection/Configuration.php]: I don't. This looks like a bug in PHPCsFixer

Comment (AltThreeTest) [inline DependencyInjection/Configuration.php]: Yeh, it's a bug with php-cs-fixer.

Comment (GrahamCampbell) [inline DependencyInjection/Configuration.php]: Yeh, it's a bug with php-cs-fixer.

Comment (stof) [inline DependencyInjection/Configuration.php]: @lsmith77 I suggest you to revert these changes (and to open an issue in the tool)

File: DependencyInjection/Configuration.php [hunk 14]
```diff
@@ -57,7 +62,7 @@ public function getConfigTreeBuilder()
                 ->arrayNode('param_fetcher_listener')
                     ->beforeNormalization()
                         ->ifString()
-                        ->then(function ($v) { return array('enabled' => in_array($v, array('force', 'true')), 'force' => 'force' === $v); })
+                        ->then(function ($v) { return ['enabled' => in_array($v, ['force', 'true']), 'force' => 'force' === $v]; })
```
Comment (stof) [inline DependencyInjection/Configuration.php]: these changes are weird though.

@GrahamCampbell is the ``symfony`` preset enforcing the short array syntax on StyleCI ? This would be weird as Symfony itself explicitly chose not to use it.

Comment (GrahamCampbell) [inline DependencyInjection/Configuration.php]: No, the symfony preset does not. Looks like this repo was run with the recommended preset, not symfony, then symfony commited afterwards...

Comment (GrahamCampbell) [inline DependencyInjection/Configuration.php]: The symfony preset doesn't specify either way.

Comment (stof) [inline DependencyInjection/Configuration.php]: @lsmith77 looks like the config you committed is not the config being used

File: DependencyInjection/Configuration.php (hunk 2 above)
Comment (lsmith77) [inline DependencyInjection/Configuration.php]: well then it will keep complaining in every PR. overall I see more benefit in style ci than in having proper indention for these two closures

File: DependencyInjection/Configuration.php (hunk 14 above)
Comment (GrahamCampbell) [inline DependencyInjection/Configuration.php]: no, it is being used - i'm saying the fixes were applied BEFORE the config was commited

Comment (lsmith77) [inline DependencyInjection/Configuration.php]: yeah .. that was propably the case

File: DependencyInjection/Configuration.php (hunk 2 above)
Comment (AltThreeTest) [inline DependencyInjection/Configuration.php]: -- Please reply above this line --

-----------------------------------------------------------
## Graham replied, on Sep 15 @ 12:45pm (UTC):

Hi there,

This is definitely an issue with php-cs-fixer. Please create an issue
on their repo.

Cheers,

Graham Campbell
support@alt-three.com


-----------------------------------------------------------
## Friendsofsymfony/fosrestbundle sent a message, on Sep 15 @ 12:44pm (UTC):

In DependencyInjection/Configuration.php:

> @@ -42,7 +42,12 @@ public function getConfigTreeBuilder()
>                  ->arrayNode('access_denied_listener')
>                      ->canBeEnabled()
>                      ->beforeNormalization()
> -                        ->ifArray()->then(function($v)
{ if (!empty($v) && empty($v['formats'])) { unset($v['enabled']); $v =
array('enabled' => true, 'formats' => $v); } return $v; })
> +                        ->ifArray()->then(function ($v)
{ if (!empty($v) && empty($v['formats'])) {
> +     unset($v['enabled']);

Yeh, it's a bug with php-cs-fixer.
-
Reply to this email directly or view it on GitHub.

-----------------------------------------------------------
Powered by Help Scout:
http://www.helpscout.net/l/main/?utm_campaign=word-of-mouth&utm_medium=email&utm_source=wom&utm_term=Alt%20Three%20Services%20Limited
